start-here/js/script-main.js

```diff
@@ -18,10 +18,70 @@ function handleSearch() {
     input.classList.add("search-box--warning");
   } else {
     warningMessage.style.display = "none";
+    fetch(`https://api.dictionaryapi.dev/api/v2/entries/en/${input.value}`)
+      .then((response) => response.json())
+      .then((data) => {
+        if (data.title) {
+          document.getElementById(
+            "title-wrapper"
+          ).innerHTML = `<p>No definition found for <strong>${input.value}</strong>.</p>`;
+        } else {
+          const meanings = data[0].meanings;
+          const phonetics = data[0].phonetics;
+
+          document.querySelector(".title").innerText = input.value;
+
+          phonetics.forEach((phonetic) => {
+            if (phonetic.text) {
+              const h2 = document.querySelector("h2");
```

Comment:
Line 36: could be better to add a `class` selector instead of the use tag `<h2>`.
It also works for this project, but considering future expansions of the project, the tag `<h2>` is too generic to be specifically related to the phonetics text.